examples/test_vault.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
<p>I guess the previous cell mistakenly says <code>create_project_secrets</code>  though the actual name is <code>create_vault_secrets</code>  ?</p><p>Anyways I don't see a reason explaining what this method is doing twice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Yes, you are correct, and yes - I'll remove redundant references.</p>